style.css

```diff
@@ -0,0 +1,131 @@
+/* Edit */
+
+h1~p {
+  color: black;
+}
+
+h1 {
+  color: black;
+}
+
+/* Varför fungerar inte detta? */
+section~h1 {
+  color: red;
+}
+/*
+Selektorn section~h1 fungerar inte eftersom den
+ specifikt riktar sig mot h1-element som följer 
+ direkt efter ett section-element som en syskonselektor.
+  Detta innebär att selektorn kommer att matcha alla h1-element
+   som är syskon till ett section-element och kommer efter det,
+    vilket inte är en vanlig struktur. 
+    */
```

Comment:
I see that you answered your own question here, you should probably clean this up.